### PR TITLE
Spring checklists corrected

### PR DESCRIPTION
**Pull request by Rommelua**

### CODE REVIEW

File: java-spring/rest-practice/jv-spring-rest-practice_checklist.md [hunk 1]
```diff
@@ -1,41 +1,41 @@
...
             }
-            /*
-              We may have method with @PostMapping("/login") too. 
-              So it's incorrect to add /register as class-level mapping.
-            */
-        }
-```
+    ```
 
 * You can return information about tickets in your ShoppingCartResponseDto / OrderResponseDto in the form of `List<Long> tiketIds`.
```
Comment (hronograf) [inline java-spring/rest-practice/jv-spring-rest-practice_checklist.md:38]:
```suggestion
* You can return information about tickets in your ShoppingCartResponseDto / OrderResponseDto in the form of `List<Long> ticketIds`.
```

File: java-spring/rest/jv-spring-rest_checklist.md [hunk 2]
```diff
@@ -1,13 +1,14 @@
...
+## Common mistakes (jv-spring-rest)
 
-* In order to properly use new dependencies in your project, you might need to run `mvn clean package`.
 * Make your code consistent. Use the @Autowired annotation everywhere with the constructors or delete it from everywhere.
 * Return `id` of entity in all response DTOs.
 * Remember about [REST resource-naming convention](https://restfulapi.net/resource-naming/).
 * In your mappers prefer setters over big constructors while converting entity to dto and vice versa.
 * Use Stream API while working with lists in your mappers.
-* In your MovieSessionRequest/ResponseDtos, we don't need to pass the whole information about Movie and CinemaHall - 
+* In your MovieSessionRequest/ResponseDtos, you don't need to pass the whole information about Movie and CinemaHall -
```
Comment (hronograf) [inline java-spring/rest/jv-spring-rest_checklist.md:8]:
```suggestion
* In your MovieSessionRequest/ResponseDtos, we don't need to pass the whole information about Movie and CinemaHall - 
```
this way we use more formal style

File: java-spring/rest/jv-spring-rest_checklist.md [hunk 3]
```diff
@@ -1,13 +1,14 @@
...
 * Make your code consistent. Use the @Autowired annotation everywhere with the constructors or delete it from everywhere.
 * Return `id` of entity in all response DTOs.
 * Remember about [REST resource-naming convention](https://restfulapi.net/resource-naming/).
 * In your mappers prefer setters over big constructors while converting entity to dto and vice versa.
 * Use Stream API while working with lists in your mappers.
-* In your MovieSessionRequest/ResponseDtos, we don't need to pass the whole information about Movie and CinemaHall - 
+* In your MovieSessionRequest/ResponseDtos, you don't need to pass the whole information about Movie and CinemaHall - 
 their ids will be enough.
+* Creating a MovieSession from RequestDto in the mapper you do not need to fetch Movie and CinemaHall from the DB by Ids. 
+Their Ids would be enough for Hibernate to save a new object with relations to DB: `movieSession.setMovie(new Movie(dto.getMovieId()));`
```
Comment (hronograf) [inline java-spring/rest/jv-spring-rest_checklist.md:11]: great addition

**Review by hronograf — APPROVED** [review]

File: java-spring/rest/jv-spring-rest_checklist.md (hunk 2 above)
Comment (Rommelua) [reply java-spring/rest/jv-spring-rest_checklist.md:8]: It seems to me, 'we' does not look consistent in this place. Are you sure with your comment?
image

Comment (hronograf) [reply java-spring/rest/jv-spring-rest_checklist.md:8]: agree, not consistent
then here it is up to you to change you->we, your->our everywhere to be more formal
or change we->you

**Pull request merged**